content/en/docs/contribute/style/style-guide.md

```diff
@@ -191,37 +194,46 @@ Set the value of `image` to nginx:1.16. | Set the value of `image` to `nginx:1.1
 Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
 {{< /table >}}
 
+However, consider quoting values where there is a risk that readers might confuse the value
+with an API kind.
+
 ## Referring to Kubernetes API resources
 
 This section talks about how we reference API resources in the documentation.
 
 ### Clarification about "resource"
 
 Kubernetes uses the word "resource" to refer to API resources, such as `pod`,
-`deployment`, and so on. We also use "resource" to talk about CPU and memory
+`deployment`, and so on. Documentation also uses "resource" to talk about CPU and memory
```

Comment:
nit: I'm confused by the formatting of pod and deployment in this section.

Reply:
I reworded; how's this?